src/values/boolean-value.js

```diff
@@ -0,0 +1,47 @@
+import FormValue from './form-value';
+import TextUtils from '../utils/text-utils';
+import NumberUtils from '../utils/number-utils';
+
+export default class BooleanValue extends FormValue {
+  constructor(element, booleanValue) {
+    super(element, booleanValue);
+
+    this.booleanValue = !!booleanValue;
+  }
+
+  format({useDisplayValue = false}) {
+    return this.booleanValue.toString;
```

Comment:
toString()

Reply:
or maybe this should have a similar check to TextValue.format.
```
if (this.element.isBoolean) {  << Is this a thing?
  return this.booleanValue;
}
return this.displayValue;
```

Reply:
Or maybe this needs a similar ternary like TextualValue:
```
return useDisplayValue ? this.displayValue : this.booleanValue;
```

Reply:
I believe format() always should return a string, so you can just return `this.displayValue`.